apps/modernization-ui/src/apps/search/SearchRouting.tsx

```diff
@@ -1,30 +1,14 @@
 import { Navigate } from 'react-router-dom';
-
-import { AdvancedSearch } from './advancedSearch/AdvancedSearch';
-
 import { SearchPage } from './SearchPage';
 import { PatientSearch } from './patient/PatientSearch';
 import { LaboratoryReportSearch } from './laboratory-report';
 import { InvestigationSearch } from './investigation';
-import { FeatureGuard } from 'feature';
 import { SimpleSearch } from './simple';
 
 const routing = [
-    {
-        path: '/advanced-search/:searchType?',
-        element: (
-            <FeatureGuard guard={(features) => !features.search.view.enabled}>
-                <AdvancedSearch />
-            </FeatureGuard>
-        )
-    },
     {
         path: 'search',
-        element: (
-            <FeatureGuard guard={(features) => features.search.view.enabled}>
```

Comment:
Question: Do we want to remove this feature flag?

Reply:
~spoilers~